snuba/datasets/discover.py

```diff
@@ -263,6 +263,8 @@ def column_expr(
         detected_dataset = detect_dataset(query, self.__transactions_columns)
 
         if detected_dataset == TRANSACTIONS:
+            if column_name == "time":
+                return self.time_expr("finish_ts", query.get_granularity(), table_alias)
```

Comment:
The order by clause on the transactions table includes start_ts not finish_ts. If we are going to group by I suspect it will be a lot more efficient doing that on the start_ts instead, any issue with that ? Same about filtering. Do we filter the discover queries based on start_ts or finish_ts ? I remember we discussed that about next/prev event time ago, But there should not be any issue in filtering by start_ts.

Reply:
This is currently just using `finish_ts` since that's what we have mapped to timestamp for all of Discover. https://github.com/getsentry/snuba/blob/master/snuba/datasets/discover.py#L271-L272. If we decide to change to "start_ts", we should probably tackle that everywhere at once and as a follow up to this.

Reply:
ok, we are likely to change the index, so we can keep finish_ts